settings.php

```diff
@@ -275,4 +275,42 @@
             get_string('option_mute_upon_entry_help', 'mod_zoom'),
             1, 1, 0);
     $settings->add($defaultmuteuponentryoption);
+
+    $invitationregexhelp = get_string('invitationregex_help', 'mod_zoom');
+    if (version_compare(normalize_version($CFG->release), '3.7.0', '<')) {
```

Comment:
Since we're using this in several places, we might as well move the check to the top of the file and save it to a boolean.

Something like:
```php
$moodlehashideif = version_compare(normalize_version($CFG->release), '3.7.0', '>=');
```

Then this could be:
```suggestion
    if (!$moodlehashideif) {
```

There is one use of this same check that predates this PR, so it would be good to update that one too.

Reply:
Good idea. I've pushed the change.